AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_PhysicalSkyAdded.py

```diff
@@ -27,6 +27,9 @@ class Tests:
     physical_sky_component = (
         "Entity has a Physical Sky component",
         "Entity failed to find Physical Sky component")
+    sky_intensity = (
+        "Sky Intensity value set to 2",
```

Comment:
I'd avoid noting specific values like this, in the event we change the intensity in the future. You can just say that the light value was updated successfully.

Reply:
fixed